setup.py

```diff
@@ -150,26 +155,26 @@ def iterRecipes(module=recipes):
 setup(
     name = APPLONGNAME,
     version = VERSION,
-    app = [APP],
```

Comment:
This and the setup_requires have caused warnings for a while.  Harmless, but also harmless to remove them.